sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -201,7 +270,8 @@ def _from_internal_entity(cls, name, internal_qd):
         # type: (str, InternalQueueDescription) -> QueueProperties
         qd = cls(
             name,
-            authorization_rules=internal_qd.authorization_rules,
+            authorization_rules=[AuthorizationRule._from_internal_entity(r) for r in internal_qd.authorization_rules] \
+                if internal_qd.authorization_rules else internal_qd.authorization_rules,
```

Comment:
nit: could internal_qd.authorization_rules be `None` instead of `[]`?, wondering if `if internal_qd.authorization_rules else []` would be better

Reply:
++Always a fan of more defensive. (going to null-coalesce though as a more terse ternary)